app/partials/actions/membership/membership-list.html

```diff
@@ -0,0 +1,50 @@
+<div class="container" ng-controller="MembershipListCtrl">
+	<h3>
+		List {{rdesc.attributes.name.singular}}
+	</h3>
+	<h5>{{rdesc.attributes.description}}</h5>
+
+	<!-- <ul>
+		<li ng-repeat="person in rolesMap">
+	      {{person.id}} {{person.role}}
+	    </li>
+</ul> -->
+
+	<div class="form-group">
+		<select chosen
+				ng-model="select_id"
+				ng-options="id as role for (id,role) in rolesMap"
```

Comment:
is ti possible to use the `details` object here and remove the need for `rolesMap` at all?